### PR TITLE
Update Lambda Cloud docs.

### PR DESCRIPTION
**Pull request by concretevitamin**
Tested (run the relevant ones):

- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: docs/source/getting-started/installation.rst [hunk 1]
```diff
@@ -101,21 +101,13 @@ Hint: run ``az account subscription list`` to get a list of subscription IDs und
...
 .. code-block:: console
 
-  $ # Create directory if required
   $ mkdir -p ~/.lambda_cloud
-  $ # Add the line "api_key = <your_api_key_here>" to lambda_keys file
   $ echo "api_key = <your_api_key_here>" > ~/.lambda_cloud/lambda_keys
 
-.. note::
-
-  Multi-node clusters and stopping instances are currently not supported on Lambda Cloud.
```
Comment (infwinston) [inline docs/source/getting-started/installation.rst:117]: Stopping instance is still not supported, right?

File: docs/source/getting-started/installation.rst [hunk 2]
```diff
@@ -101,21 +101,13 @@ Hint: run ``az account subscription list`` to get a list of subscription IDs und
 Lambda Cloud
 ~~~~~~~~~~~~~~~~~~
 
-Lambda Labs GPU Cloud, or Lambda Cloud, is a cloud provider offering low-cost GPUs. You can learn more about them `here <https://lambdalabs.com/>`__.
-
-To configure Lambda Cloud access, go to the `API Keys <https://cloud.lambdalabs.com/api-keys>`_ page on your Lambda console to generate a key and then add it to :code:`~/.lambda_cloud/lambda_keys` by running:
+`Lambda Cloud <https://lambdalabs.com/>`__ is a cloud provider offering low-cost GPUs. To configure Lambda Cloud access, go to the `API Keys <https://cloud.lambdalabs.com/api-keys>`_ page on your Lambda console to generate a key and then add it to :code:`~/.lambda_cloud/lambda_keys`:
```
Comment (romilbhardwaj) [inline docs/source/getting-started/installation.rst:104]: Single underscore? 
```
`Lambda Cloud <https://lambdalabs.com/>`_
```

**Review by romilbhardwaj — APPROVED** [review]
LGTM! Haven't checked rendering though, but please verify.

**Review by infwinston — APPROVED** [review]
Small comment on stopping instance, otherwise looks good!

File: docs/source/getting-started/installation.rst (hunk 1 above)
Comment (concretevitamin) [reply docs/source/getting-started/installation.rst:117]: We don't have to mention that early in the installation, I think. There's appropriate messages from the code.

File: docs/source/getting-started/installation.rst (hunk 2 above)
Comment (concretevitamin) [reply docs/source/getting-started/installation.rst:104]: TIL rst's ambiguity, both are supported. Done.

**Pull request merged**